app/helpers/admin/enterprises_helper.rb

```diff
@@ -39,7 +39,6 @@ def enterprise_side_menu_items(enterprise)
         { name: 'inventory_settings', icon_class: "icon-list-ol", show: is_shop },
         { name: 'tag_rules', icon_class: "icon-random", show: is_shop },
         { name: 'shop_preferences', icon_class: "icon-shopping-cart", show: is_shop },
-        { name: 'users', icon_class: "icon-user", show: true }
```

Comment:
Was this line removed on purpose? Because the "Users" tab doesn't show up anymore.

Reply:
no this was intentional, there's some angular there that's messing with the page URL. Will restore it once I merge this[remove angular from users](https://github.com/openfoodfoundation/openfoodnetwork/pull/9729) into here.

Thank You for the code suggestions.

Reply:
Ah, I see. Thanks for sharing your PR. Is there another PR's related to this one?

Reply:
There's this one https://github.com/openfoodfoundation/openfoodnetwork/pull/9761 but I just merged it into this PR itself in my latest commit.